se.data/langs/es_la.json

```
@@ -0,0 +1,171 @@
+  "help_cmd_copy": "l Copiar selección actual al portapapeles",
+  "help_cmd_brush_flood": "l Registra brocha para realizar un llenado de inundación con [block] empezando en el bloque seleccionado",
+  "more_moves_redo": "w Valor demasiado grande, rehaciando todas las acciones de %s",
+  "no_undo_history": "w Sin historia de acciones para mostrar de %s",
+  "action_outline": "contorno",
+  "help_cmd_brush_feature": "l Registra brocha para colocar decoraciones",
+  "action_stack": "apilado",
+  "help_cmd_brush_info": "l Da información detallada de la brocha en mano",
+  "help_header_prefix": "c ----------------- [ ",
+  "help_selected_wand": "c Varita seleccionada",
+  "action_prism_polygon": "polígono_prisma",
+  "action_line": "línea",
+  "action_structure_paste": "pegado_estructura",
+  "brush_list_header": "bc === Las brochas actuales son ===",
+  "help_cmd_brush_cylinder": "l Registra brocha para crar un cilindro con [radius] y [height] a lo largo de [axis] de [block]",
+  "help_cmd_brush_spray": "l Registra brocha para pintura en aerosol con el radio y la cantidad de puntos dados",
+  "more_moves_undo": "w Valor demasiado grande, deshaciando todas las acciones de %s",
+  "help_welcome_tooltip": "y Síii!",
+  "action_item_angel": "item de bloque ángel",
+  "saved_structure": "w Estructura guardada como %s.nbt",
+  "help_cmd_rotate": "l Rota [deg] en torno a [pos]",
+  "brush_info_action": "b  Acción: ",
+  "help_cmd_brush_pyramid": "l Registra brocha para crear pirámide con [radius] y [height] sobre eje [axis] en la dirección del signo",
+  "langs_completeness": "gi Nota: %s la traducción está completa en un %s%%: %s frases faltantes",
+  "no_undo": "r Sin acciones que deshacer de %s",
+  "help_cmd_brush_outline": "l Registra brocha de contorno",
+  "help_cmd_rotate_tooltip": "g <axis> (eje) debe ser x, y o z",
+  "angel_block_none_error": "r Aún no hay bloque ángel definido! Usar 'angel new' para registrar uno",
+  "help_cmd_move": "l Mueve la selección a pos <pos>",
+  "langs_changed": "gi Idioma cambió a %s.",
+  "copy_force": "ri Sobreescribiendo portapapeles anterior con el nuevo",
+  "action_ellipsoid": "elipsoide",
+  "help_cmd_brush_cuboid": "l Registra brocha para crear un cuboide de dimensiones [x] [y] y [z] de [block]",
+  "entry_undo_1": "w %d: tipo: %s (dimensión: %s):",
+  "brush_reach": "w El alcance de las brochas es %d bloques",
+  "help_cmd_undo_history": "l Muestra la historia de acciones deshechas",
+  "success_redo": "gi Se rechicieron %d exitosamente, rellenando %d bloques",
+  "no_longer_brush": "w %s ya no es más una brocha",
+  "help_cmd_undo": "l Deshace los últimos <n> movimientos, uno sólo por defecto",
+  "brush_info_flags": "b Banderas: ",
+  "action_sphere": "esfera",
+  "help_cmd_paste": "l Pega portapapeles",
+  "help_cmd_brush_generic": "l Sostiene un ítem para covnertirlo en brocha",
+  "angel_block_bad_item": "r Bloque nagel no puede ser alg que ya está asignado a otra acción",
```

```suggestion
  "angel_block_bad_item": "r Bloque ángel no puede ser algo que ya está asignado a otra acción",
```
And maybe subjunctive (`esté`).